Client/Application/LeanplumIntegration.swift

```diff
@@ -114,11 +115,14 @@ class LeanplumIntegration {
             profile?.prefs.setBool(!canInstallFocus(), forKey: PrefsKeys.HasFocusInstalled)
         }
 
+        let appVersion = Float(Bundle.main.object(forInfoDictionaryKey: "CFBundleShortVersionString") as! String)
```

Comment:
I think you should this be 'as! Number', as Leanplum has told me it needs to be sent as a number for > comparisons to work. Here are the LP docs: https://www.leanplum.com/docs/ios/users#user-attributes